keyboards/skyloong/gk980/bl/bl.c

```diff
@@ -0,0 +1,36 @@
+//  Copyright 2023 JZ-Skyloong (@JZ-Skyloong)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+
+void suspend_power_down_kb() {
+    writePinLow(MAC_PIN);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb() {
+    suspend_wakeup_init_user();
+}
+
+bool shutdown_kb(bool jump_to_bootloader) {
+    return shutdown_user(jump_to_bootloader);
+}
+
+layer_state_t default_layer_state_set_kb(layer_state_t state) {
```

Comment:
Doesnt call _user function

Reply:
I see .Thanks for your guidance.

Reply:
You need to actually fix the issue, and not just mark it as resolved.

Reply:
I update it . Could you please review it again? Thank you.

Reply:
You havent changed anything? It still doesnt call `default_layer_state_set_user` anywhere.

Reply:
Would you please check it again? @zvecr